pages/index.jsx

```diff
@@ -9,9 +34,21 @@ export default function Home() {
       </Head>
 
       <main>
-        <h1>
-          Hello World!
-        </h1>
+        <h1>Week of April 18 to April 25, 2022</h1>
+
+        <LoadingStatus
+          loadingStatus={loadingStates[loadingStatus]}
```

Comment:
With the other `loadingState` changes, you can just passing `loadingStatus` here.